### PR TITLE
Update Croatian translation

### PR DESCRIPTION
**Pull request by milotype**
- minor changes/corrections
- complete translation
- removed unused strings

Is there anything else which needs to be translated? I found: [joplin/packages/lib/locales/hr_HR.json](url), but I don't know if this is being translated automatically ...

Also, please add some information about "localisation/internationalisation" in the general description of the app ... for instance, by adding the term **multilingual** with a link ... Maybe something like:

_Joplin is a free, <a href="https://joplinapp.org/#localisation">multilingual</a>, open source note taking and to-do application, which can handle a large number of notes organised into notebooks._

### CODE REVIEW

File: packages/tools/locales/hr_HR.po [hunk 1]
```diff
@@ -3923,254 +3955,3 @@ msgstr "Označava zadatak završenim."
...
-#~ "`sync.2.path` to specify the target directory."
-#~ msgstr ""
-#~ "Meta sinkronizacije. U slučaju sinkroniziranja s vlastitim datotečnim "
-#~ "sustavom, postavi `sync.2.path` na ciljani direktorij."
-
-#~ msgid "To-do title:"
-#~ msgstr "Naslov zadatka:"
-
-#~ msgid "\"%s\": \"%s\""
-#~ msgstr "\"%s\": \"%s\""
```
Comment (tessus) [inline packages/tools/locales/hr_HR.po:4176]: Please do not delete these. We keep old translations around. The reason being, aource strings might be removed bu6t latre added again. In that case, we already have a proper translation (if it was translated in the past).

**Comment by tessus** [comment]
Wow, thanks a lot for the extensive translation.

> Also, please add some information about "localisation/internationalisation" in the general description of the app ... for instance, by adding the term multilingual with a link ...
> Joplin is a free, multilingual, open source note taking and to-do application, which can handle a large number of notes organised into notebooks.

This has to be decided by @laurent22. I think he has not added it before, because being multilingual is almost a given these days. But this is just a guess. IMO we could add it, but I'm not so sure about the link.
As I said the decision lies with Laurent.

**Comment by milotype** [comment]
Thank you for your answer ...

I disagree about keeping unused strings ... I suggest to allow the deletion of unused strings. If they get used again in future, it is sipmle for translators to translate them and to adjust them to the current translation, because translation of strings/terms may change over time ...

IMO, internationalisation/multilinguality of apps is not given these days at all ... in my experience, there is often a machine translation of the app's website, with no implication of the translation of the app itself.

**Comment by tessus** [comment]
> I disagree about keeping unused strings

Laurent will have to decide. The same is true for https://github.com/laurent22/joplin/pull/4496#issuecomment-775234349

I can only say what I've already said. We keep them around for a reason.

**Comment by laurent22** [comment]
Like @tessus said, we keep the old translations. @milotype other than the deleted translation issue, the pull request is nearly ready to merge, so please let us know if you can complete it.

**Comment by milotype** [comment]
@laurent22 So do you want me to simply copy and paste those old strings back into my translation file?

**Comment by laurent22** [comment]
You should be using poedit to be honest, as it will do the right thing automatically, including keeping old translations. Manually pasting strings is not a good idea as if you get a quote or newline off, the file will end up broken.

**Comment by tessus** [comment]
@milotype how did you remove them? I guess you just deleted them from the file. Thus the only way is to copy them back in.

**Comment by milotype** [comment]
@laurent22 Sorry, but that's too much to ask for ... I do use PoEdit, but I'd have to go through all 775 strings again ...

@tessus I deleted them through the PoEdit's function for removing obsolete strings. If you insist on having those old/fuzzy/wrong translations, the easiest way would be to simply paste them at the end of file with BBEdit ...)

My Croatian translation in this pull request is up to date and complete (100 %). The current Croatian translation is only 25 % translated (https://github.com/laurent22/joplin#localisation)!

IMO, having 100 % translated (without old and often fuzzy and wrong translations) is definitely better than having only 25 % translated (with a few old and often fuzzy and wrong translations)!

Please don't feel offended … it's just my opinion as an experienced translator ...

**Comment by tessus** [comment]
> Please don't feel offended … it's just my opinion as an experienced translator ...

I don't, but unless you commit to take care of that translation forever, we want the "old" translations in there. They might not be important to you, but might be to others, who will translate stuff in the future.
So yeah, pleast just paste them back in and we can merge.

**Comment by tessus** [comment]
You can just add another commit and push to your branch.

**Comment by tessus** [comment]
Thanks

**Pull request merged**